code/drasil-docLang/Drasil/Sections/TableOfSymbols.hs

```diff
@@ -21,16 +20,22 @@ table st ls f
       Table [atStart symbol_, atStart description, atStart' unit_]
       (mkTable [P . (`symbol` st), f, toSentence] filteredChunks)
       (titleize tOfSymb) True
-    | otherwise = error $ "Same symbols for different quantities found "  ++ show symUidPairDuplicates
+    | otherwise = error $ errorMessage 
     where 
         filteredChunks = filter (`hasStageSymbol`st) ls
         symbolsCol     = map (`symbol` st) filteredChunks
         uidCol         = map (view uid)    filteredChunks
         symUidPair     = zip symbolsCol uidCol
-        symDuplicates  = symbolsCol \\ nub symbolsCol
+        symDuplicates  = nub (symbolsCol \\ nub symbolsCol)
         noDuplicate    = null symDuplicates
-        symUidPairDuplicates = [(first (render . symbolDoc) x, snd x) | x <- symUidPair, 
-          fst x `elem` symDuplicates]
+        -- If there are duplicates then the following will extract the UID's of duplicates symbols
+        extractPairs symb = filter (\x -> fst x == symb) symUidPair
+        extractUid  = map (\x -> snd x)
+        extractUidFromPairs = text . show . extractUid . extractPairs
+        errSymUidDuplicates = vcat $ map (\symb -> 
+          extractUidFromPairs symb<+>text "all have symbol"<+>symbolDoc symb) symDuplicates
+        errorMessage = "Same symbols for different quantities found: " ++ render errSymUidDuplicates
+
```

Comment:
This is as clean as I could get it.

The following is the new error message, say that we have 2 examples sharing the same symbol and 3 examples with another symbol, then the error will print different lines for each duplicated symbols. Like the following:
Same symbols for different quantities found: ["dimMax","dimMin"] all have symbol d_max
["cWeightMin","sdMax","sdMin"] all have symbol w_min

Reply:
Good.